unliftio/test/UnliftIO/PooledAsyncSpec.hs

```diff
@@ -0,0 +1,191 @@
+{-#LANGUAGE DeriveDataTypeable#-}
+
+module UnliftIO.PooledAsyncSpec (spec) where
+
+import Test.Hspec
+import Control.Concurrent
+import Data.List (nub, sort)
```

Comment:
I'd like to request the removal of all uses of `nub` because it is O(n²).

When the tests pass, n is very small so it is not an actual program in that case, but I'm trying to discourage complete disentivisation of `nub` throughout all Haskell projects, so it would be nice if we could lead by example in `unliftio`.

https://github.com/nh2/haskell-ordnub provides easily copy-pastable alternatives, which are also as `nubOrd` in `rio` and `stack`.

Starting from LTS-13 (current nightly), you can also use `Data.Containers.ListUtils.nubOrd` from `containers`.

Reply:
This has been resolved.